rotkehlchen/chain/ethereum/modules/makerdao/constants.py

```diff
@@ -9,3 +32,40 @@
 CPT_VAULT = 'makerdao vault'
 CPT_DSR = 'makerdao dsr'
 CPT_MIGRATION = 'makerdao migration'
+
+
+# This dict is needed as standalone to be used for proxies tokens detection
+UNIQUE_TOKENS_COLLATERAL_TYPES_MAPPING = {
+    'BAT-A': A_BAT,
+    'KNC-A': A_KNC,
+    'TUSD-A': A_TUSD,
+    'USDC-A': A_USDC,
+    'USDT-A': A_USDT,
+    'WBTC-A': A_WBTC,
+    'ZRX-A': A_ZRX,
+    'MANA-A': A_MANA,
+    'PAXUSD-A': A_PAX,
+    'COMP-A': A_COMP,
+    'LRC-A': A_LRC,
+    'LINK-A': A_LINK,
+    'BAL-A': A_BAL,
+    'YFI-A': A_YFI,
+    'GUSD-A': A_GUSD,
+    'UNI-A': A_UNI,
+    'RENBTC-A': A_RENBTC,
+    'AAVE-A': A_AAVE,
+    'MATIC-A': A_MATIC,
+}
+
+
+# Add collateral types for eth and for tokens that have multiple collateral types
+ALL_COLLATERAL_TYPES_MAPPING = {
+    'ETH-A': A_ETH,
+    'ETH-B': A_ETH,
+    'ETH-C': A_ETH,
+    'USDC-B': A_USDC,
```

Comment:
The thing is that there are multiple modes for USDC and WBTC. In the previous implementation I had to do `set()` in order to get rid of the duplicates. Now I made these dicts so that in the dict with tokens for proxies there are only unique tokens.

Does it make sense or is better to keep all tokens in one dict and then do `set()`?